packages/js-sdk/src/sandbox/filesystem/index.ts

```diff
@@ -141,53 +129,61 @@ export class Filesystem {
     return res.data
   }
 
-  /**
-   * Writes content to a file on the path.
-   *   When writing to a file that doesn't exist, the file will get created.
-   *   When writing to a file that already exists, the file will get overwritten.
-   *   When writing to a file that's in a directory that doesn't exist, you'll get an error.
-   *
-   * @param path Path to a new file. For example '/dirA/dirB/newFile.txt' when creating 'newFile.txt'
-   * @param data Data to write to a new file
-   * @param opts Options for the request
-   * @returns Information about the written file
-   */
+  async write(path: string, data: WriteData, opts?: FilesystemRequestOpts): Promise<EntryInfo>
+  async write(files: WriteEntry[], opts?: FilesystemRequestOpts): Promise<EntryInfo[]>
   async write(
-    path: string,
-    data: string | ArrayBuffer | Blob | ReadableStream,
+    pathOrFiles: string | WriteEntry[],
+    dataOrOpts?: WriteData | FilesystemRequestOpts,
     opts?: FilesystemRequestOpts
-  ): Promise<EntryInfo> {
-    const blob = await new Response(data).blob()
+  ): Promise<EntryInfo | EntryInfo[]> {
+    if (typeof pathOrFiles === 'string' && Array.isArray(dataOrOpts)) {
+      throw new Error('Cannot specify path with array of files')
```

Comment:
Let's change this error string to be more explicit what user is doing wrong here:
```
Cannot specify both path and array of files.
```

maybe we could be even more helpful here:
```
Cannot specify both path and array of files. You have to specify either path and data for a single file or an array for multiple files.
```